pages/search.js

```diff
@@ -300,7 +300,8 @@ class SearchPage extends React.Component {
     return (
       <Page title="Search" showSearch={false}>
         <Container
-          backgroundImage="url(/static/images/search-background.png)"
+          backgroundImage="url(/static/images/home/fiscalhost-blue-bg-lg.png)"
+          style={{ transform: 'rotate(180deg)' }}
```

Comment:
What is the purpose of the rotation there?

Reply:
Note that this image originally was intended for the homepage,

![image](https://user-images.githubusercontent.com/12435965/174827033-ea569f1e-462b-4f37-89cd-50ad16154c75.png)

and as such it is inverted. We need to rotate it by 180 degrees for it to be used in the `/search` page.

Reply:
I find that approach a bit hacky but ok.

Reply:
Alternative maybe is to store a duplicate of this image in the correct rotation set. But I didn't do that as that introduces duplication of a file unnecessary. 🤔